### PR TITLE
fix(Magento2): bug fixes

### PR DESCRIPTION
**Pull request by omryYotpo**
#YO-315
#YO-314
#YO-317
#YO-319
#YO-318

### CODE REVIEW

File: Yotpo/Yotpo/etc/adminhtml/system.xml [hunk 1]
```diff
@@ -20,11 +20,11 @@
                     <p>Click <a href="http://support.yotpo.com/entries/22635391-advanced-magento-features" target="_blank" 
                       title="yotpo">here</a> for the advanced customization guide. 
                     </p><br/>
-                    <p style="font-size:18px;">Dont have an App Key? <a style="font-size:18px;" href="http://my.yotpo.com/" target="_blank" 
+                    <p style="font-size:18px;">Dont have an App Key? <a style="font-size:18px;" href="http://www.yotpo.com/" target="_blank"
```
Comment (vladshub) [inline Yotpo/Yotpo/etc/adminhtml/system.xml]: @omryYotpo Why is the link for www and not for my.yotpo.com/register ?

Comment (omryYotpo) [inline Yotpo/Yotpo/etc/adminhtml/system.xml]: @NetanelMoshkovitz  what do you say about Vlad's note?

Comment (omryYotpo) [inline Yotpo/Yotpo/etc/adminhtml/system.xml]: @billykov Can you make a call about this?

Comment (billykov) [inline Yotpo/Yotpo/etc/adminhtml/system.xml]: @omryYotpo vlad is correct - my.yotpo.com/register

**Comment by vladshub** [comment]
@NetanelMoshkovitz this looks good...

**Comment by omryYotpo** [comment]
@vladshub Fix for a bug that when the api is down make sure that we don't disrupt the order change status

File: Yotpo/Yotpo/etc/adminhtml/system.xml (hunk 1 above)
Comment (vladshub) [inline Yotpo/Yotpo/etc/adminhtml/system.xml]: !fix @omryYotpo what is the status of this bug?

Comment (omryYotpo) [inline Yotpo/Yotpo/etc/adminhtml/system.xml]: @vladshub This is fixed already (we are looking at an outdated diff here)

Comment (vladshub) [inline Yotpo/Yotpo/etc/adminhtml/system.xml]: !fixed

**Pull request merged**